amf-cli/shared/src/test/scala/amf/validation/AMFModelAssertionTest.scala

```diff
@@ -452,4 +452,9 @@ class AMFModelAssertionTest extends AsyncFunSuite with Matchers {
       getFirstRequestPayload(bu).annotations.lexical() shouldBe PositionRange(Position(13, 6), Position(15, 0))
     }
   }
+
```

Comment:
Couldn't this be a valid model test ?

Reply:
yes but I need to check that it conforms, and the modelAssertion method discards the results

Reply:
hmm i didn't understand. The UniquePlatformValidationTest* or ValidModel* see if the parsing and validation report conform

Reply:
moved the test to UniquePlatformValidationTest